spec/01-unit/013-reports_spec.lua

```diff
@@ -28,6 +30,8 @@ describe("reports", function()
       assert.matches("hello=world", res, nil, true)
       assert.matches("signal=stub", res, nil, true)
       assert.matches("baz=bat", res, nil, true)
+      assert.matches("foobar=" .. cjson.encode({ foo = "bar" }), res, nil, true)
+      assert.matches("bazbat=" .. cjson.encode({ baz = "bat" }), res, nil, true)
```

Comment:
It seems like `cjson` is `nil` in this test, causing the CI to fail.

Reply:
Yes. Bad monkey-patching. Passing now.